src/events/qaMention.js

```diff
@@ -1,9 +1,13 @@
 const { Events } = require('discord.js');
 const { DISCORD_CONFIG } = require('../config');
+const { pollResults } = require('../utils/poll-vote');
 
 module.exports = {
   name: Events.MessageCreate,
```

Comment:
Maybe we could explore instead of using MessageCreate, using MessageUpdate. Idk if poll message is updated when it ended, so we probably could access to more properties and also we could possibly get the result even in DRAWS.

This could be handle on another PR since this PR is all based on messageCreate event and the embed that appears when poll end

Reply:
I created the task https://github.com/TeamNovaSoft/discord-bot/issues/30